resources/nl/cemu.po

```diff
@@ -200,6 +305,178 @@ msgstr "Herstarten"
+
+#: ChecksumTool.cpp
+msgid "Export successful"
+msgstr "Succesvol geëxporteerd"
+
+#: ChecksumTool.cpp
+msgid "Can't write to file: {}"
+msgstr "Kan niet schrijven naar bestand: {}"
+
+#: ChecksumTool.cpp
+msgid "JSON file doesn't satisfy needed schema"
+msgstr "JSON bestand voldoet niet aan het verplichte formaat"
+
+#: ChecksumTool.cpp
+msgid "Wrong title id: {}"
+msgstr "Verkeerde titel ID:"
+
+#: ChecksumTool.cpp
+msgid "Wrong version: {}"
+msgstr "Verkeerde versie: {}"
+
+#: ChecksumTool.cpp
+msgid "Wrong region: {}"
+msgstr "Verkeerde regio: {}"
+
+#: ChecksumTool.cpp
+msgid "The verification data doesn't include a wud hash!"
+msgstr "De verificatie data heeft geen wud controlesom!"
+
+#: ChecksumTool.cpp
+msgid ""
+"Your game image is invalid!\n"
+"\n"
+"Your hash:\n"
+"{}\n"
+"\n"
+"Expected hash:\n"
+"{}"
+msgstr ""
+"Jouw game rom is niet geldig!\n"
+"\n"
+"Jouw controlesom:\n"
+"{}\n"
+"\n"
+"Verwachte controlesom:\n"
+"{}"
+
+#: ChecksumTool.cpp
+msgid "Select a file to export the errors"
+msgstr "Selecteer een bestand om de errors naar te exporteren"
+
+#: ChecksumTool.cpp
+msgid "Can't open file to write!"
+msgstr "Kan bestand niet openmaken om naar te schrijven!"
+
+#: ChecksumTool.cpp
+msgid "The following files are missing:"
+msgstr "De volgende bestanden ontbreken:"
+
+#: ChecksumTool.cpp
+msgid ""
+"{} files are missing!\n"
+"Do you want to export a list of them to a file?"
+msgstr ""
+"{} bestanden ontbreken!\n"
+"Wil je een lijst exporteren hiervan exporteren naar een bestand?"
+
+#: ChecksumTool.cpp
+msgid ""
+"{} files have an invalid hash!\n"
+"Do you want to export a list of them to a file?"
+msgstr ""
+"{} bestanden hebben een ongeldige controlesom!\n"
+"Wil je een lijst hiervan exporteren naar een bestand?"
+
+#: ChecksumTool.cpp
+msgid ""
+"Multiple issues with your game files have been found!\n"
+"Do you want to export them to a file?"
+msgstr ""
+"Meerdere problemen zijn gedetecteerd met je game bestanden!\n"
+"Wil je deze exporteren naar een bestand?"
+
+#: ChecksumTool.cpp
+msgid "Your game files are valid"
+msgstr "Je game bestanden zijn geldig"
+
+#: ChecksumTool.cpp
+msgid "JSON parse error: {}"
+msgstr "JSON parseerfout: {}"
+
+#: ChecksumTool.cpp
+msgid "Can't open file!"
+msgstr "Kan bestand niet openen!"
+
+#: ChecksumTool.cpp
+msgid "Can't parse json file!"
+msgstr "Kan json bestand niet parseren!"
+
+#: ChecksumTool.cpp
+msgid "Open checksum entry"
+msgstr "Open controlesom item"
+
+#: ChecksumTool.cpp
+msgid "Grabbing game files"
+msgstr "Lijst maken van game bestanden"
+
+#: ChecksumTool.cpp
+msgid "Hashing game file: {}/{}"
+msgstr "Controlesom maken van bestand: {}/{}"
+
+#: ChecksumTool.cpp
+msgid "Generated checksum of {} game files"
+msgstr "Controlesom gemaakt voor {} game bestanden"
+
+#: ChecksumTool.cpp
+msgid "Reading game image: {}"
+msgstr "Lezen van game schijfkopie: {}"
+
+#: ChecksumTool.cpp
+msgid "Reading game image: {}/{}kb"
+msgstr "Lezen game schijfkopie: {}/{}kb"
```

Comment:
Is it intended that here "Reading game image" is translated in a slightly different way compared to the previous string?
(Also, it would be better if there were a space between `{}` and `kb`)